### DISCUSSION TITLE
B站长音源为什么只能播放第一个？

### DISCUSSION THREAD

**Discussion by LIMINGNING** [maotoumao/MusicFree#301] [General]
在播放B站的“许嵩歌曲140首精选【分集+歌词+高音质”显示有9小时32分钟50秒，但是因为他这个音源是分集的，在使用MusicFree播放的时候只能播放第一个视频的音乐，我该如何解决这个问题？

**Comment by Radignis** [comment]
从专辑点进去

**Comment by lldmw13** [comment]
请问解决了么，评论的这个从专辑点进去不明白什么意思，点了专辑就提示复制成功。
